manifests/bosh-manifest/deployments/020-compiled-package-cache.yml

```diff
@@ -2,10 +2,7 @@ jobs:
 - name: bosh
   properties:
     compiled_package_cache:
-      provider: s3
```

Comment:
I do like to be explicit, even if by default is s3. This makes it more obvious for any other developers and it would make it easier to parse with an automation tool or similar , without having to check the default in the spec.